spec/fixtures.rb

```diff
@@ -0,0 +1,136 @@
+class User
+  include Virtus.model(finalize: false)
```

Comment:
Virtus is cool but I don't think we really make use of it here. How about a simple
```ruby
class Model
  def initialize(params = {})
    params.each { |p, k| instance_variable_set("@#{p}", k) }
  end
end

# and then
class User
  attr_accessor :id, :name, :email, :tweets, :friends, :messages
end
```

Keep in mind that this gem is made to work with POROs, so I think tests should reflect that.